gruntfile.js

```diff
@@ -6,14 +6,14 @@
 'use strict';
 
 const Umberto = require( 'umberto' );
+const webpackReactConf = require( './react/webpack.config.js' );
 
 module.exports = function( grunt ) {
 	const packageVersion = grunt.file.readJSON( 'package.json' ).version;
 
 	require( 'load-grunt-tasks' )( grunt );
-
-	grunt.loadNpmTasks( 'grunt-contrib-connect' );
```

Comment:
Why did you remove this task? It will break the `docs-serve` task.

Reply:
It shouldn't as this task is listed in dev deps and `load-grunt-tasks` loads tasks from all dependencies.